### ISSUE TITLE
is component dead?

### ISSUE THREAD

**Issue by thisconnect** [component/component.github.io#87]

**Comment by chemzqm** [comment]
I'm using components of component to make website and new components from npm everyday, but I think the website and command line tool of component is dead,  webpack and npm are better choice IMO.

**Comment by thisconnect** [comment]
thanks. I like the small repos on (https://github.com/component). 

http://component.github.io/ is confusing. seems to be different libraries than what is on  github.com/component

**Comment by timaschew** [comment]
// Update

Just using the word **component** is ambiguous!

Component, the package manager and build tool, located under the [componentjs](https://github.com/componentjs) organization is deprecated, see here: https://github.com/componentjs/component/blob/master/Readme.md or here
https://github.com/componentjs/component/issues/639

All the repos within the [component](https://github.com/component)  organization are kind of the official components. 

http://component.github.io provides a registry and everyone can add his library to this registry, like npm. You just need a `component.json` in your project root and then you can [publish it this way](https://github.com/componentjs/guide/blob/master/creating-components/publishing.md)

**Comment by thisconnect** [comment]
Ty for the quick reply. So componentjs is DEPRECATED. The repos on https://github.com/component have recent commits.

What is your opinion on official components?

**Comment by stephenmathieson** [comment]
I'm still maintaining a few of the "official components", but only for usage in npm. This said, I strongly encourage you to use webpack or browserify rather than component.

**Comment by thisconnect** [comment]
Yes For building I use rollup for es6 and sometimes use webpack or broserify. I find the official components small focused and cute. So I wondered if you use them. Should they be rewritten to es2015 one day?

**Comment by stephenmathieson** [comment]
> Should they be rewritten to es2015 one day?

probably not, as component doesn't support clever lifecycle hooks like npm's `prepublish`.. plus they work fine as es5 currently ;)
